jerry-core/ecma/operations/ecma-bigint.c

```diff
@@ -197,7 +206,11 @@ ecma_bigint_parse_string_value (ecma_value_t string, /**< ecma string */
   JERRY_ASSERT (ecma_is_value_string (string));
 
   ECMA_STRING_TO_UTF8_STRING (ecma_get_string_from_value (string), string_buffer_p, string_buffer_size);
-  ecma_value_t result = ecma_bigint_parse_string (string_buffer_p, string_buffer_size, options);
+
+  const lit_utf8_byte_t *nonws_start_p = ecma_string_trim_front (string_buffer_p, string_buffer_p + string_buffer_size);
+
+  lit_utf8_size_t nonws_size = (lit_utf8_size_t) ((string_buffer_p + string_buffer_size) - nonws_start_p);
+  ecma_value_t result = ecma_bigint_parse_string (nonws_start_p, nonws_size, options);
```

Comment:
Perhaps we could do this in `ecma_bigint_parse_string` since the end white space check is there as well.